hack/verify-all.sh

```diff
@@ -60,7 +60,7 @@ if $SILENT ; then
 fi
 
 # remove protobuf until it is part of direct generation
-EXCLUDE="verify-godeps.sh verify-generated-protobuf.sh"
+EXCLUDE="verify-godeps.sh verify-godep-licenses.sh verify-generated-protobuf.sh"
```

Comment:
It's not super fast, but I figured if we weren't checking godeps there wasn't much point in checking something that depends on the godeps...